src/app/modules/project/overview/overview.component.html

```diff
@@ -19,7 +19,7 @@
 
 <h1>Search project</h1>
 <div class="row">
-  <div class="col-2">
+  <div class="col-lg-3">
```

Comment:
This makes the filter bigger on desktop-size as well. Something that is not wanted. Compare dex.software on full screen compared to your branch (project overview page). You will notice that there's less room for the projects on your branch because the filter is taking an additional collumn.
![image](https://user-images.githubusercontent.com/15904543/93791659-cca3b080-fc34-11ea-8d6d-41ccdb9abe63.png)